keyboards/m60_split/config.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright 2021 alin elena <alin@elena.space>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBABA
+#define PRODUCT_ID      0x0006
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    alin elena
+#define PRODUCT         m60 split
+
+/* key matrix size */
+#define MATRIX_ROWS 10 // Rows are doubled-up
+#define MATRIX_COLS 6
+
+#define MATRIX_COL_PINS { B5, B7, B8, A0, A7, B10 }
+#define MATRIX_ROW_PINS { B11, B0, B1, A2, A3 }
+#define MATRIX_COL_PINS_RIGHT { B14, A10, A15, B3, B4, B5 }
+#define MATRIX_ROW_PINS_RIGHT { B1, B0, B10, B11, A7 }
+
+//#define LED_NUM_LOCK_PIN B12
+//#define LED_SCROLL_LOCK_PIN B13
+//#define LED_CAPS_LOCK_PIN C13
+
+//#define ENCODERS_PAD_A { A8 }
+//#define ENCODERS_PAD_B { A9 }
+
+//#define ENCODER_RESOLUTION 4
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+#define MATRIX_IO_DELAY 5
+
+#define SPLIT_HAND_PIN B9
+#define SOFT_SERIAL_PIN B6
+#define SELECT_SOFT_SERIAL_SPEED 0
+
+#define RGB_DI_PIN B15
+
+#define RGBLED_NUM 24
+#define RGBLED_SPLIT { 12, 12 }
+#define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```